projects/kit/directives/dropdown-selection/dropdown-selection.directive.ts

```diff
@@ -70,7 +70,7 @@ export class TuiDropdownSelectionDirective
     private range?: Range;
 
     @Input()
-    set tuiDropdownSelection(handler: TuiBooleanHandler<Range> | undefined) {
+    set tuiDropdownSelection(handler: TuiBooleanHandler<Range> | '' | undefined) {
```

Comment:
I wonder if `undefined` here is meant for that `''` mistakenly

Reply:
fixed